modules/common/src/test/scala/surge/kafka/KafkaPartitionShardRouterActorSpec.scala

```diff
@@ -129,10 +136,10 @@ class KafkaPartitionShardRouterActorSpec
 
       val newPartitionAssignments = Map[HostPort, List[TopicPartition]](hostPort1 -> List(partition0, partition1, partition2), hostPort2 -> List())
 
-      partitionProbe.send(routerActor, TracedMessage(PartitionAssignments(newPartitionAssignments), Map[String, String]()))
```

Comment:
theres a default of Map.empty for the headers arg now, so  Map[String, String]() is no longer needed